medusa/scripts/medusa_wrapper.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import subprocess
+import os
+
+
+def main():
```

Comment:
**Issue:** This is not working as expected. The calls to medusa-wrapper seem to return instantly and the job behind it is not executed. In the following traces, I create a backup `b1` individually on all nodes, then create a `b2` backup using the `medusa backup-cluster` orchestration. It returns successfully but no backup is created.

```
[2024-04-15 10:03:20,748] INFO: Updating backup index
[2024-04-15 10:03:21,106] INFO: Backup done
[2024-04-15 10:03:21,106] INFO: - Started: 2024-04-15 10:03:12
                        - Started extracting data: 2024-04-15 10:03:13
                        - Finished: 2024-04-15 10:03:21
[2024-04-15 10:03:21,106] INFO: - Real duration: 0:00:07.475006 (excludes time waiting for other nodes)
[2024-04-15 10:03:21,107] INFO: - 136 files, 109.63 KB
[2024-04-15 10:03:21,107] INFO: - 136 files copied from host (136 new, 0 reuploaded)
[2024-04-15 10:03:21,107] INFO: - 0 kept from previous backup (b1)

ubuntu@ip-172-31-21-127:~$ medusa backup-cluster --backup-name=b2
[2024-04-15 10:03:50,418] INFO: Resolving ip address
[2024-04-15 10:03:50,420] INFO: ip address to resolve 172.31.21.127
[2024-04-15 10:03:50,421] INFO: Monitoring provider is noop
[2024-04-15 10:03:50,436] INFO: Using credentials CensoredCredentials(access_key_id=A..K, secret_access_key=*****, region=us-west-2)
[2024-04-15 10:03:50,436] INFO: Connecting to s3_us_west_oregon with args {}
[2024-04-15 10:03:50,642] INFO: Starting backup b2
[2024-04-15 10:03:50,769] INFO: Resolving ip address 172.31.21.127
[2024-04-15 10:03:50,769] INFO: ip address to resolve 172.31.21.127
[2024-04-15 10:03:50,770] INFO: Resolving ip address 172.31.21.127
[2024-04-15 10:03:50,770] INFO: ip address to resolve 172.31.21.127
[2024-04-15 10:03:50,771] INFO: Resolving ip address 172.31.23.245
[2024-04-15 10:03:50,771] INFO: ip address to resolve 172.31.23.245
[2024-04-15 10:03:50,772] INFO: Resolving ip address 172.31.31.16
[2024-04-15 10:03:50,772] INFO: ip address to resolve 172.31.31.16
[2024-04-15 10:03:50,774] INFO: Creating snapshots on all nodes
[2024-04-15 10:03:50,775] INFO: Executing "nodetool -Dcom.sun.jndi.rmiURLParsing=legacy snapshot -t medusa-b2" on following nodes ['ip-172-31-21-127.us-west-2.compute.internal', 'ip-172-31-23-245.us-west-2.compute.internal', 'ip-172-31-31-16.us-west-2.compute.internal'] with a parallelism/pool size of 500
[2024-04-15 10:03:55,307] INFO: Job executing "nodetool -Dcom.sun.jndi.rmiURLParsing=legacy snapshot -t medusa-b2" ran and finished Successfully on all nodes.
[2024-04-15 10:03:55,307] INFO: A snapshot medusa-b2 was created on all nodes.
[2024-04-15 10:03:55,307] INFO: Uploading snapshots from nodes to external storage
[2024-04-15 10:03:55,307] INFO: Executing "mkdir -p /tmp/medusa-job-fbc450d8-2ae9-4979-8e9a-64d409fa9877; cd /tmp/medusa-job-fbc450d8-2ae9-4979-8e9a-64d409fa9877 && medusa-wrapper sudo medusa  -vvv backup-node --backup-name b2   --mode differential" on following nodes ['ip-172-31-21-127.us-west-2.compute.internal', 'ip-172-31-23-245.us-west-2.compute.internal', 'ip-172-31-31-16.us-west-2.compute.internal'] with a parallelism/pool size of 1
[2024-04-15 10:03:58,323] INFO: Job executing "mkdir -p /tmp/medusa-job-fbc450d8-2ae9-4979-8e9a-64d409fa9877; cd /tmp/medusa-job-fbc450d8-2ae9-4979-8e9a-64d409fa9877 && medusa-wrapper sudo medusa  -vvv backup-node --backup-name b2   --mode differential" ran and finished Successfully on all nodes.
[2024-04-15 10:03:58,323] INFO: A new backup b2 was created on all nodes.
[2024-04-15 10:03:58,324] INFO: Backup duration: 7.823342
[2024-04-15 10:03:58,324] INFO: Backup of the cluster done.

ubuntu@ip-172-31-21-127:~$ medusa list-backups
[2024-04-15 10:04:08,442] INFO: Resolving ip address
[2024-04-15 10:04:08,443] INFO: ip address to resolve 172.31.21.127
[2024-04-15 10:04:08,458] INFO: Using credentials CensoredCredentials(access_key_id=A..K, secret_access_key=*****, region=us-west-2)
[2024-04-15 10:04:08,459] INFO: Connecting to s3_us_west_oregon with args {}
b1 (started: 2024-04-15 10:02:33, finished: 2024-04-15 10:03:21)
```

Reply:
@adejanovski , thanks for catching this. I've spent some time orchestrating this.
One scenario where I could get this situation (~orchestration succesful, backup absent), was when I ran:
```
medusa --config-file medusa.ini backup-cluster --backup-name=b11
```
The issue was that the orchestrator crafted a command to run, which featured just `medusa.ini` (so a path relative to $CWD). It then ran in `/tmp/medusa-jobid`, and the ini file wasn't there. This failed, but we didn't properly propagate the return code upwards.

This is fixed now and this particular failure reaches the orchestrator, which errors out.

I was able to run a cluster-backup if I ran:  
```
medusa --config-file /full/path/to/medusa.ini backup-cluster --backup-name=b11
```
Another option woud be to put the path to the ini file into the ini file of the orchestrator, which should then propagate into the workers.